js/scripts.js

```diff
@@ -142,6 +137,26 @@ exampleModal.addEventListener('show.bs.modal', (event) => {
   document.getElementById('popup_tech').innerHTML = listTech;
 });
 
-menuOpen();
-menuClose();
-loadProjects();
\ No newline at end of file
+// form-validation//
+
+const form = document.querySelector('#contactForm');
+const email = document.querySelector('#email');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const emailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  const emailVal = email.value.trim();
+
+  if (!emailRegex.test(emailVal)) {
+    document.getElementById('alertWin').style.display = 'block';
+
+    email.focus();
+  } else {
+    form.submit();
+  }
```

Comment:
- [ ] Kindly make sure that when a user submits the contact form with the email field containing uppercase characters an error message should show up. Like `Only lowercase allowed for email` or `Use only lowercase in email field`.
- [ ] Kindly make sure that the error message should show up close to the "submit" button.
To understand this better you can view this loom video given [here](https://www.loom.com/share/2b59121265f54e10903f8d402b088870).